tests/test_playbooks/vars/server.yml.example

```diff
@@ -10,6 +10,9 @@ foreman_proxy: "foreman.example.com"
 # Parameter for snapshot test
 snapshot_host_name: "test_host"
 
+# Parameter for job invocation  test
+foreman_host: "foreman.example.com"
```

Comment:
I think this variable needs something like this: https://github.com/theforeman/foreman-ansible-modules/blob/develop/tests/vcr_python_wrapper.py#L128

So configuring the value to another host does not break the test.

Reply:
thanks for the point, I added a matcher into vcr_python_wrapper